types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
@RyanCavanaugh Transpiled code does not have default export anymore after its converted to cjs module. The ES6 default export is overridden and thus this import won't work anymore, returning `undefined`. The changed exports/imports are indeed correct at this point.